packages/teams-js/src/public/liveShare.ts

```diff
@@ -0,0 +1,404 @@
+     * @param event Name of event.
+     * @param listener Function to receive event.
+     */
+    on(event: string, listener: (...args: unknown[]) => void);
+
+    /**
+     * Registers to receive an event only once.
+     * @param event Name of event.
+     * @param listener Function to receive event.
+     */
+    once(event: string, listener: (...args: unknown[]) => void);
+
+    /**
+     * Un-registers a previously registered event handler.
+     * @param event Name of event.
+     * @param listener Function previously registered.
+     */
+    off(event: string, listener: (...args: unknown[]) => void);
+
+    /**
+     * Provides the current connected state of the container
+     */
+    readonly connectionState: unknown;
+
+    /**
+     * A container is considered **dirty** if it has local changes that have not yet been acknowledged by the service.
+     * You should always check the `isDirty` flag before closing the container or navigating away from the page.
+     * Closing the container while `isDirty === true` may result in the loss of operations that have not yet been
+     * acknowledged by the service.
+     *
+     * A container is considered dirty in the following cases:
+     *
+     * 1. The container has been created in the detached state, and either it has not been attached yet or it is
+     * in the process of being attached (container is in `attaching` state). If container is closed prior to being
+     * attached, host may never know if the file was created or not.
+     *
+     * 2. The container was attached, but it has local changes that have not yet been saved to service endpoint.
+     * This occurs as part of normal op flow where pending operation (changes) are awaiting acknowledgement from the
+     * service. In some cases this can be due to lack of network connection. If the network connection is down,
+     * it needs to be restored for the pending changes to be acknowledged.
+     */
+    readonly isDirty: boolean;
+
+    /**
+     * Whether the container is disposed, which permanently disables it.
+     */
+    readonly disposed: boolean;
+
+    /**
+     * The collection of data objects and Distributed Data Stores (DDSes) that were specified by the schema.
+     * These data objects and DDSes exist for the lifetime of the container.
+     */
+    readonly initialObjects: Record<string, unknown>;
+
+    /**
+     * The current attachment state of the container.  Once a container has been attached, it remains attached.
+     * When loading an existing container, it will already be attached.
+     */
+    readonly attachState: string;
+
+    /**
+     * A newly created container starts detached from the collaborative service.
+     * Calling `attach()` uploads the new container to the service and connects to the collaborative service.
+     *
+     * @remarks This should only be called when the container is in the
+     * {@link @fluidframework/container-definitions#AttachState.Detatched} state.
```

Comment:
Does this @link actually resolve now that no types are being included from this package?

Reply:
probably not... good catch. I'll look those over.

Reply:
Removed namespace per discussion